classes/Product.php

```diff
@@ -5702,7 +5702,7 @@ public static function getProductProperties($id_lang, $row, Context $context = n
         $unitPriceRatio = self::computeUnitPriceRatio($row, $id_product_attribute, $quantity, $context);
         $row['unit_price_ratio'] = $unitPriceRatio;
         $row['unit_price_tax_excluded'] = $unitPriceRatio != 0 ? $priceTaxExcluded / $unitPriceRatio : 0.0;
```

Comment:
I think the price tax excluded could also be modified to use `$row['unit_price']` directly it will reduce the problems related with float precision